services/core-web/src/components/Home.js

```diff
@@ -80,10 +83,15 @@ export class Home extends Component {
 
   loadStaticContent = () => {
     this.props.dispatch(showLoading());
-    const staticContentActionCreators = Object.getOwnPropertyNames(staticContent).filter(
-      (property) => typeof staticContent[property] === "function"
-    );
-    staticContentActionCreators.forEach((action) => this.props.dispatch(staticContent[action]()));
+    console.log("HELPS");
```

Comment:
Remove console.log, and is commented-out code below needed?

Reply:
removed.